### PR TITLE
fix: ensure compatability with werkzeug 3.0.0

### PR DESCRIPTION
**Pull request by LiadOz**
I have disabled the internal cookies that the test client uses. When use_cookies is True, it overrides the cookie header that is passed with open_kwargs to the open function, so after setting use_cookies to False, there is no need to change the cookies. This is tested with in `tests/test_cookies.py`.
I took the liberty of adding a github workflow and removing old versions from setup.cfg, let me know if this should be changed.

### CODE REVIEW

**Comment by LiadOz** [comment]
@vmalloc can you review this PR?

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 requests
 Flask
+six
```
Comment (vmalloc) [inline requirements.txt:3]: Why is this dependency needed?

Comment (vmalloc) [reply requirements.txt:3]: N/M - found it

**Comment by vmalloc** [comment]
@LiadOz Sure, sorry that it slipped underneath the radar within all my notifications.

Can you explain what the problem is and why this actually fixes it? Thanks in advance!

**Comment by LiadOz** [comment]
The cookie_jar attribute was removed from the Werkzeug Client class. So it can't be used anymore. However, it's not really needed, since we already forward the `headers` of the request, the cookies should already be used, so that code wasn't actually needed.
I tested it with `tests/test_cookies.py` and it works as expected.

**Comment by vmalloc** [comment]
Is it backwards compatible with older versions of Werkzeug/flask?

**Comment by LiadOz** [comment]
Werkzeug versions prior to 1.0.0 could have issues, it introduced some changes with cookies handling and I tested using that version. Should I add a version marker for it?

**Comment by vmalloc** [comment]
At least worth constraining the version so that the conflict is clear to whomever installs it then

**Comment by LiadOz** [comment]
> At least worth constraining the version so that the conflict is clear to whomever installs it then

done

**Comment by vmalloc** [comment]
Thanks @LiadOz !

**Comment by LiadOz** [comment]
@vmalloc Are you planning to release a new version of flask-loopback to pypi? I can set up a release job if you need.

**Comment by vmalloc** [comment]
Hi! That would be greatly appreciated if you're up for it

**Pull request merged**